bundles/org.openhab.binding.shelly/src/main/resources/OH-INF/i18n/shelly.properties

```diff
@@ -92,7 +92,7 @@ thing-type.shelly.shellyplusrgbw.description = Shelly Plus RGBW PM - LED Control
 # Plus Devices
 thing-type.shelly.shellyplus1.description = Shelly Plus 1 (Single Relay Switch)
 thing-type.shelly.shellyplus1pm.description =  Shelly Plus 1PM - Single Relay Switch with Power Meter
-thing-type.shelly.shellyplus2-relay.description = Shelly Plus 2PM - Dual Relay Switch with Power Meter
+thing-type.shelly.shellyplus2pm-relay.description = Shelly Plus 2PM - Dual Relay Switch with Power Meter
```

Comment:
Why this change? I guess you would also need to change this line then:
https://github.com/openhab/openhab-addons/blob/9f73acaf39bc206a6e0020e050567b5ed147a657/bundles/org.openhab.binding.shelly/src/main/resources/OH-INF/thing/shellyGen2_relay.xml#L38

Reply:
you're right, fixed the xml